### PR TITLE
Entry Screen created

### LINKED ISSUE #28: Create Entry Screen
**What are the tasks that need to be done?**
- [x] Created Entry Screen
- [x] Add User Name
- [x] Add User avatar(s) 
- [x] Add User Role
- [x] Add Settings Button

### In-Depth Feature Description
Reusable Component for each User. 

**Dependencies:**
- Top Bar Component from #23 
- Player Avatar Component from #20 

### Images for Reference:
image

### PR DESCRIPTION
**Pull request by Jmcl168**
# Description
This PR fixes #28 

EntryScreen component created using TopBarSettings component and Player component

# Demo
(Please include screenshots/GIF here if applicable)

Before: N/A

After:   
entry

# Type of change:
* New feature (Non-breaking change that adds functionality)

# Checklist:
* Have you merged main into your branch?
* Have you tested your changes to ensure it works as expected and does not break existing functionality?
* If applicable, please ensure sufficient tests are added that is related to the changes.
* Please ensure this PR has a `label`, is linked to an `issue` and is related to a `project`
* If new documentation is required for this change, have you created a new documentation issue that describes the documentation needed?

### CODE REVIEW

File: frontend/src/Styles/Player.module.css [hunk 1]
```diff
@@ -4,9 +4,10 @@
     border-radius: 50%;
     background-color: rgb(43, 43, 43);
     color: white;
-    display: flex;
+    display: inline-flex;
```
Comment (Jmcl168) [inline frontend/src/Styles/Player.module.css:7]: @ateagit please let me know if this change breaks your player component. I did this to allow for multiple player components that adjust themselves to fit the screen depending on the number present.

Comment (ateagit) [reply frontend/src/Styles/Player.module.css:7]: Yep, should be fine. I'll make a seperate pr for table if it breaks.

**Review by ateagit — APPROVED** [review]
LGTM

**Comment by reeved** [comment]
Merging since has been approved.

**Pull request merged**